src/VisualStudio/CSharp/Impl/Options/Formatting/FormattingOptionPageControl.xaml

```diff
@@ -25,6 +25,27 @@
                     <CheckBox x:Name="FormatOnPasteCheckBox" x:Uid="FormatOnPasteCheckBox" />
                 </StackPanel>
             </GroupBox>
+            <GroupBox x:Name="FormatDocumentSettingsGroupBox" x:Uid="FormatDocumentSettingsGroupBox" >
+                <StackPanel x:Uid="StackPanel_2">
+                    <CheckBox x:Name="AreCodeCleanupRulesConfiguredCheckBox" x:Uid="AreCodeCleanupRulesConfiguredCheckBox" Visibility="Hidden" Height="0"/>
+                    <CheckBox x:Name="AllCSharpFormattingRulesCheckBox" x:Uid="AllCSharpFormattingRulesCheckBox" IsEnabled="False" IsChecked="True"/>
```

Comment:
I think you don't need this option? user can't change it anyway.

Reply:
This checkbox is called out (always checked, not allow to change) in @kuhlenh 's design note.